lib/utils/gender.ts

```diff
@@ -6,14 +6,15 @@ import { Applicant } from '../../domain/HousingApi';
  * @returns {string}
  */
 export const getGenderName = (applicant: Applicant): string => {
-  const gender = applicant.person?.gender?.toString();
```

Comment:
Throw in a simple test here to make sure it's default is still returing correctlly.

Reply:
Perhaps better here to drop default to under case to keep same functionality.